blades/abstract_blade.h

```diff
@@ -109,6 +111,47 @@ class AbstractBlade : public BladeBase, public SaberBase {
     return GetPrimaryBlade() == this;
   }
 
+  // Blaster Effects
```

Comment:
This long list of identical functions makes me think that there has to be a better way to do this.
Probably outside the scope of this change though...

Reply:
Yeah I figured based on the growing list I saw this going the way of SABERFUN but was not about to make the assumption of doing it.